ployst/github/test/test_end_to_end.py

```diff
@@ -15,7 +15,7 @@ class TestEndToEnd(TestCase):
     def setUp(self):
         ensure_dummy_clone_available()
 
-    @patch(__name__ + '.tasks.hierarchy.client', MockClient())
+    @patch('ployst.github.tasks.hierarchy.client', MockClient())
```

Comment:
`mock` was failing to patch this. I'm also confused as to how it worked, since `__name__` will be the name of this test module, not the source one...?